src/EnergyPlus/UnitarySystem.cc

```diff
@@ -15978,112 +15763,6 @@ namespace UnitarySystems {
         this->m_HeatRecoveryMassFlowRate = HeatRecMassFlowRate;
     }
 
-    void UnitarySys::heatPumpRunFrac(Real64 const PLR,   // part load ratio
```

Comment:
Runtime fraction now calculated by coil objects. This function isn't needed in parent objects anymore.